testing/widgets/datagrid/VirtualScrolling.test.js

```diff
@@ -0,0 +1,27 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+
+fixture('DataGrid.VirtualScrolling')
+  .page('http://localhost:8080/')
+  .beforeEach(async (t) => {
+    await t
+      .resizeWindow(900, 600);
+  });
+
+runManualTest(test, 'DataGrid', 'VirtualScrolling', 'jQuery', (test) => {
+  test('VirtualScrolling', async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await t.hover($('.dx-scrollbar-vertical .dx-scrollable-scroll'));
+    await takeScreenshot('datagrid_virtual_scrolling_2_desktop.png');
+
+    await t.scrollBy('.dx-scrollable-container', 0, 250);
```

Comment:
Change 250 to 2 000 000 to scroll to the second part of content as it was before with drag.